src/lasdi/latent_space.py

```diff
@@ -72,6 +94,9 @@ def __init__(self, physics, config):
         self.space_dim = np.prod(self.qgrid_size)
         hidden_units = config['hidden_units']
         n_z = config['latent_dimension']
+        threshold = config["threshold"] if "threshold" in config else 0.1
+        value = config["value"] if "value" in config else 0.0
+        num_heads = config['num_heads'] if 'num_heads' in config else 1
```

Comment:
@Eisenl10 , can we move line 97-99 into the if statement in line 119-127, exactly where they are used? So that people would know these variables are used only for these special cases.